packages/coordinator/src/screens/AuthScreens/ResetPasswordScreen.tsx

```diff
@@ -1,33 +1,75 @@
 import { useState } from "react";
 import styles from "./AuthScreens.module.scss";
 import Input from "../../components/Input";
-import Button from "../../components/Button";
+import Button, { ButtonVariant } from "../../components/Button";
 import logoImage from "./../../assets/blood-bank-zichron-logo.svg";
+import popupImage from "../../assets/popup_resetpass.png";
 import chevronSvg from "./../../assets/icons/chevron-right-small.svg";
 import { useNavigate } from "react-router-dom";
+import { Popup } from "../../components/Popup/Popup";
 
 export interface ResetPasswordScreenProps {
   onResetPassword: (
     email: string,
     emailError: (error: string) => void
-  ) => Promise<void>;
+  ) => Promise<boolean>;
 }
 
 export default function ResetPasswordScreen(props: ResetPasswordScreenProps) {
   const [email, setEmail] = useState("");
   const [emailError, setEmailError] = useState("");
   const [isLoading, setIsLoading] = useState(false);
 
+  const [popupOpen, setPopupOpen] = useState<boolean>(false);
+  const [popupError, setPopupError] = useState<string | undefined>(undefined);
+
   const navigate = useNavigate();
 
   const resetPassword = async () => {
     setIsLoading(true);
-    await props.onResetPassword(email, setEmailError);
+    const res = await props.onResetPassword(email, setEmailError);
     setIsLoading(false);
+
+    return res;
+  };
+
+  const handleResetPass = async () => {
+    const resetStatus = await resetPassword();
+    setPopupOpen(true);
+
+    if (!resetStatus) {
+      setPopupError("כתובת המייל שלך לא נמצאה");
+    }
   };
 
+  const popupContent = (
+    <div className={styles.resetPassPopupContent}>
+      <img src={popupImage} alt="" className={styles.resetPassPopupImg} />
+      <div className={styles.resetPassPopupTitle}>
+        {popupError ? "אופס" : "מייל איפוס בדרך אליך"}
+      </div>
+      <div className={styles.resetPassPopupMessage}>
+        {popupError ?? "ברגעים הקרובים יישלח אליך מייל עם לינק לאיפוס סיסמא"}
+      </div>
+      <Button
+        className={styles.resetPasswordScreenPopupButton}
+        onClick={() => {
+          setEmail("");
+          setPopupOpen(false);
+        }}
+        title={popupError ? "נסה/י שנית" : "לא קיבלתי, שלחו לי שוב"}
+        variant={popupError ? ButtonVariant.contained : ButtonVariant.text}
+      />
+    </div>
+  );
+
   return (
     <div className={styles.screen}>
+      <Popup
```

Comment:
Why are you using the same popup for both messages? I find this confusing and hard to extend in the future.

Consider this alternative:
1. Create 2 popups with built in content and title, one for each reason.
2. Create 2 state variables - showSuccessPopup and showFailurePopup
3. Call `resetPassword`. If it succeeded (i.e promise returned without throwing anything) - show success popup. Otherwise, show failure.

Reply:
done